pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/promtail/config.go

```diff
@@ -138,7 +138,7 @@ MemorySwapMax=0
 Restart=always
 RestartSec=5
 EnvironmentFile=/etc/environment
-ExecStartPre=/bin/sh -c "systemctl set-environment HOSTNAME=$(hostname)"
+ExecStartPre=/bin/sh -c "systemctl set-environment HOSTNAME=$(echo -n $(hostname) | tr '[:upper:]' '[:lower:]')"
```

Comment:
```suggestion
ExecStartPre=/bin/sh -c "systemctl set-environment HOSTNAME=$(hostname | tr [:upper:] [:lower:])"
```

I think that you do not need the `echo` and the single quotes. I did not run the code but tested with this command
```bash
/bin/sh -c "echo -n $(hostname | tr [:upper:] [:lower:])"
```